src/supermarket/app/controllers/api/v1/cookbook_uploads_controller.rb

```diff
@@ -128,6 +129,12 @@ def destroy_version
 
     begin
       authorize! @cookbook, :destroy?
+      raise VersionMustExist if @cookbook.cookbook_versions.count == 1
+    rescue VersionMustExist
+      error({ error_code: t('api.error_codes.conflict'),
+              error_messages: [t('api.error_messages.only_cookbook_version')],
+              error: t('api.error_messages.only_cookbook_version') },
```

Comment:
I have not yet found what client might be expecting a collection of `error_messages` (which I copied from other error returns here in the API), but `error` is the message displayed by the `knife` CLI. So added it here to make the error meaningful to an end user attempting to use `knife supermarket` or `knife cookbook site` to delete the last version of a cookbook.